src/solutions/booking_businesses/request.rs

```diff
@@ -0,0 +1,117 @@
+        body: true
+    );
+    get!(
+        doc: "Get bookingBusinesses from solutions",
+        name: list_booking_businesses,
+        path: "/bookingBusinesses"
+    );
+    get!(
+        doc: "Get the number of the resource",
+        name: booking_businesses,
+        path: "/bookingBusinesses/$count"
+    );
+}
+
+impl BookingBusinessesIdApiClient {
+    api_client_link_id!(staff_member, StaffMembersIdApiClient);
+    api_client_link_id!(customer, CustomersIdApiClient);
+    api_client_link!(appointments, AppointmentsApiClient);
+    api_client_link!(services, ServicesApiClient);
+    api_client_link_id!(appointment, AppointmentsIdApiClient);
+    api_client_link_id!(service, ServicesIdApiClient);
+    api_client_link!(custom_questions, CustomQuestionsApiClient);
+    api_client_link!(customers, CustomersApiClient);
+    api_client_link_id!(custom_question, CustomQuestionsIdApiClient);
+    api_client_link!(staff_members, StaffMembersApiClient);
+
+    delete!(
+        doc: "Delete navigation property bookingBusinesses for solutions",
+        name: delete_booking_businesses,
+        path: "/bookingBusinesses/{{RID}}"
+    );
+    get!(
+        doc: "Get bookingBusinesses from solutions",
+        name: get_booking_businesses,
+        path: "/bookingBusinesses/{{RID}}"
+    );
+    patch!(
+        doc: "Update the navigation property bookingBusinesses in solutions",
+        name: update_booking_businesses,
+        path: "/bookingBusinesses/{{RID}}",
+        body: true
+    );
+    post!(
+        doc: "Create new navigation property to calendarView for solutions",
+        name: create_calendar_view,
+        path: "/bookingBusinesses/{{RID}}/calendarView",
+        body: true
+    );
+    get!(
+        doc: "Get calendarView from solutions",
+        name: list_calendar_view,
+        path: "/bookingBusinesses/{{RID}}/calendarView"
+    );
+    get!(
+        doc: "Get the number of the resource",
+        name: calendar_view,
+        path: "/bookingBusinesses/{{RID}}/calendarView/$count"
+    );
+    delete!(
+        doc: "Delete navigation property calendarView for solutions",
+        name: delete_calendar_view,
+        path: "/bookingBusinesses/{{RID}}/calendarView/{{id}}",
+        params: booking_appointment_id
+    );
+    get!(
+        doc: "Get calendarView from solutions",
+        name: get_calendar_view,
+        path: "/bookingBusinesses/{{RID}}/calendarView/{{id}}",
+        params: booking_appointment_id
+    );
+    patch!(
+        doc: "Update the navigation property calendarView in solutions",
+        name: update_calendar_view,
+        path: "/bookingBusinesses/{{RID}}/calendarView/{{id}}",
+        body: true,
+        params: booking_appointment_id
+    );
+    post!(
+        doc: "Invoke action cancel",
+        name: cancel,
+        path: "/bookingBusinesses/{{RID}}/calendarView/{{id}}/cancel",
+        body: true,
+        params: booking_appointment_id
+    );
```

Comment:
[Calendar view](https://github.com/sreeise/graph-rs-sdk/blob/329201426c15b36c3b685d50ae15c59ee1753694/src/users/calendar_view/request.rs) has its own code generation under the users folder. We should use a link to this client as it has all the same methods.

You will need to add calendarView to the filterPath call in codegen for BookingBusinessesIdApiClient and add an import for calendarView and then add an ApiClientLink in codgen for linking to calendarView as a method call.

Reply:
That's been done. Got caught out again with adding the link-ups in the wrong match but should all be good now :)